content/tutorials/publish-subscribe.textile

```diff
@@ -14,17 +14,68 @@ Messages published can contain string, JSON object, JSON array or binary data pa
 
 <%= partial 'tutorials/_step-1-setup-free-account' %>
 
-h2. Step 2 - Install Ably
+h2. 
+  default: Step 2 – Install Ably
+  android: Step 2 – Set up environment and install Ably
 
 blang[java].
+  To start using Ably in your Java application, you need to include the Ably Client library. We recommend that you include the latest client library via Gradle in your project's @gradle.build@ file.
+  
+  ```[groovy]
+  apply plugin: 'application'
+  mainClassName = 'io.ably.tutorial.Example'
+
+  repositories {
+    jcenter()
+      maven { url "https://raw.github.com/paddybyers/Java-WebSocket/mvn-repo/" }
```

Comment:
Note that I am not going to make the same comments on this as I did with the history commit 7fe61663e2eff8c22e7e52907f05a286ba0007c8.  Please make sure all changes are applied here